mlflow/utils/docstring_utils.py

```diff
@@ -267,6 +267,29 @@ class that describes the model's inputs and outputs. If not specified but an
 - `summarization <https://huggingface.co/transformers/main_classes/pipelines.html#transformers.SummarizationPipeline>`_
 - `text2text-generation <https://huggingface.co/transformers/main_classes/pipelines.html#transformers.Text2TextGenerationPipeline>`_
 - `text-generation <https://huggingface.co/transformers/main_classes/pipelines.html#transformers.TextGenerationPipeline>`_
+"""
+        ),
+        "save_pretrained": (
+            """If set to ``False``, MLflow will not save the Transformer model weight files,
+instead only saving the reference to the HuggingFace Hub model repository and its commit hash.
+This is useful when you load the pretrained model from HuggingFace Hub and want to log or save
+it to MLflow without modifying the model weights. In such case, specifying this flag to
+``False`` will save the storage space and reduce time to save the model.
+
+.. warning::
```

Comment:
Let's file a follow-up to add this warning and an end-to-end example to the transformers Guide (as well as adding a quick tutorial notebook showing the behavior of logging a few iterations of saving models without saving the weights, followed by a selection of the 'most promising one' to log the weights artifacts and register the model).

Reply:
Absolutely! Created and added to next sprint: ML-38531